src/Language/Writer/IntValueWriter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Digia\GraphQL\Language\Writer;
+
+use Digia\GraphQL\Language\AST\Node\IntValueNode;
+use Digia\GraphQL\Language\AST\Node\NodeInterface;
+
+class IntValueWriter extends AbstractWriter
+{
+    /**
+     * @param NodeInterface|IntValueNode $node
+     * @inheritdoc
+     */
+    public function write(NodeInterface $node): string
+    {
+        return $node->getValue();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function supportsWriter(NodeInterface $node): bool
```

Comment:
Should we consider prefix it with `Is`? `IsSupportsWriter`?

Reply:
This is similar to other namings, I guess we can change it later.